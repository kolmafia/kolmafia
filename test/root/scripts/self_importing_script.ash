import <self_importing_script.ash>

print("This line should only be printed once.");
