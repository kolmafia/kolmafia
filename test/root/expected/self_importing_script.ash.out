This line should only be printed once.
